composer.json

```diff
@@ -7,11 +7,11 @@
     "license": "MIT",
     "description": "The officially supported client for Postmark (http://postmarkapp.com)",
     "require": {
-        "php": ">=7.0.0",
+        "php": ">=8",
```

Comment:
Can you be sure that this lib will run without any problems on PHP 8.3, or 9.0, or 10.0? Surely this should be `~8.0 || ~8.1 || ~8.2`

Reply:
Thanks I will update this.